bigbone/src/main/kotlin/social/bigbone/api/entity/CredentialAccount.kt

```diff
@@ -180,6 +180,12 @@ data class CredentialAccount(
     @SerialName("following_count")
     val followingCount: Long = 0,
 
+    /**
+     * Domains of websites allowed to credit the account.
+     */
+    @SerialName("attribution_domains")
+    val attributionDomains: List<String> = emptyList(),
```

Comment:
How do you feel about nullability vs `emptyList()` as a default? As this is only added in 4.4, for older versions we’d always return this as `emptyList()` if the JSON doesn’t contain it. 🤔 

Maybe we should at the very least add a note on when this was added so that consumers know what to expect?

```suggestion
     * Domains of websites allowed to credit the account.
     * @since Mastodon 4.3.0
     */
    @SerialName("attribution_domains")
    val attributionDomains: List<String> = emptyList(),
```

Reply:
I hadn't actually put too much thought into this. The empty list default was based on the understanding that all attributes not marked as **required** by Mastodon may be missing from a response, and that we typically try to have some non-null default value in those cases.

Either way would work for me, I just don't see how this attribute is special in any way.